Automatic Validation ended with: 
> 2026-03-27T21:51:08.8680577Z ##[error] Manifest Error: Schema header not found. File: 1b360695961b9b81f584e4d057964ddb47b5cf4c35d8f3ec441fecd5110c7414 Manifest Error: Schema header not found. File: 41d2707aef135f46cbe66b08a1e380e99eae9d4d615fd53f9245f54840dd737e Manifest Error: Schema header not found. File: 4e0e9f130a782e942e5b32ac11d23221a5c733d6039801ef7823eac32abfa6b4

(Automated response - build 1240.)

--- a/manifests/0/0-don/clippy/1.5.8/0-don.clippy.installer.yaml
+++ b/manifests/0/0-don/clippy/1.5.8/0-don.clippy.installer.yaml
@@ -1,3 +1,5 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
 PackageIdentifier: 0-don.clippy
 PackageVersion: 1.5.8
 InstallerType: nullsoft

--- a/manifests/0/0-don/clippy/1.5.8/0-don.clippy.locale.en-US.yaml
+++ b/manifests/0/0-don/clippy/1.5.8/0-don.clippy.locale.en-US.yaml
@@ -1,3 +1,5 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
 PackageIdentifier: 0-don.clippy
 PackageVersion: 1.5.8
 PackageLocale: en-US

--- a/manifests/0/0-don/clippy/1.5.8/0-don.clippy.yaml
+++ b/manifests/0/0-don/clippy/1.5.8/0-don.clippy.yaml
@@ -1,3 +1,5 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.9.0.schema.json
+
 PackageIdentifier: 0-don.clippy
 PackageVersion: 1.5.8
 DefaultLocale: en-US

